keith.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "keithhchen",
+    "email": "keithhchen@icloud.com"
+  },
+  "records": {
+    "CNAME": "https://keithchen2025.vercel.app"
```

CNAME can't contain protocol
```suggestion
    "CNAME": "keithchen2025.vercel.app"
```